package.json

```diff
@@ -23,6 +26,9 @@
     "styled-components": "^6.0.8"
   },
   "devDependencies": {
+    "@testing-library/jest-dom": "^6.4.2",
+    "@testing-library/react": "^14.2.2",
+    "@types/jest": "^29.5.12",
```

Comment:
why do you need this type package?

Reply:
While trying to write and run this test, one of the tutorials suggested installing this package to solve a TS error in `vite.config.js`.
![Screenshot 2024-04-11 at 14 41 01](https://github.com/gatinhap/plant-app-ver_0/assets/72195350/65c1a377-1fd4-4fd3-8d59-9d4010cc7588)

Reply:
hm, defineConfig accepts either an object or a function, maybe you can try with the function approach. We don't use that package and it works. Minor issue but let's try

Reply:
With the function approach it worked and the TS error is not showing anymore so I removed @types/jest library